src/evidently/suite/base_suite.py

```diff
@@ -227,7 +231,7 @@ def show(self, mode="auto"):
         try:
             from IPython.display import HTML
 
-            return HTML(self._render(determine_template(mode), template_params))
+            return HTML(self._render(inline_iframe_html_template, template_params))
```

Comment:
we are using `iframe` (not like just embed all js and html stuff as it is) in order to render UI in jupyter notebook